### ISSUE TITLE
version 1.6.1: segmentation error

### ISSUE THREAD

**Issue by giulio1945** [sm0svx/svxlink#398]
svxlink when connecting with RemoteTrx v1.2.0 from this error:SvxLink v1.6.1 Copyright (C) 2003-2017 Tobias Blomberg / SM0SVX

Using configuration file: /etc/svxlink/svxlink.conf
--- Using sample rate 48000Hz

Starting logic: SimplexLogic
Loading RX: Rx1
Loading TX: Tx1
Loading module "ModuleHelp" into logic "SimplexLogic"
	Found /usr/lib/x86_64-linux-gnu/svxlink/ModuleHelp.so
	Module Help v1.0.0 starting...
Loading module "ModuleParrot" into logic "SimplexLogic"
	Found /usr/lib/x86_64-linux-gnu/svxlink/ModuleParrot.so
	Module Parrot v1.1.1 starting...
Loading module "ModuleEchoLink" into logic "SimplexLogic"
	Found /usr/lib/x86_64-linux-gnu/svxlink/ModuleEchoLink.so
	Module EchoLink v1.4.1 starting...
Loading module "ModuleTclVoiceMail" into logic "SimplexLogic"
	Found /usr/lib/x86_64-linux-gnu/svxlink/ModuleTcl.so
	Module Tcl v1.0.1 starting...
SimplexLogic: Event handler script successfully loaded.

Starting logic: ReflectorLogic
ReflectorLogic: Connecting to 192.168.1.40:5300
ReflectorLogic: Disconnected from 192.168.1.40:5300: Connection refused
EchoLink directory status changed to ON
--- EchoLink directory server message: ---
EchoLink Server v2.5.9997

ECHO6: N.Virginia, USA

ReflectorLogic: Connecting to 192.168.1.40:5300
ReflectorLogic: Connection established to 192.168.1.40:5300
*** ERROR[ReflectorLogic]: Unpacking failed for TCP message header
ReflectorLogic: Disconnected from 192.168.1.40:5300: Locally ordered disconnect
segmentation error

remotetrx response:
RemoteTrx v1.2.0 Copyright (C) 2003-2015 Tobias Blomberg / SM0SVX

RemoteTrx comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
welcome to redistribute it in accordance with the terms and conditions in the
GNU GPL (General Public License) version 2 or later.

Using configuration file: /etc/svxlink/remotetrx.conf
Setting up trx "NetUplinkTrx"
RX: Rx1
TX: Tx1

NetUplinkTrx: Client connected: 192.168.1.251:55278
NetUplinkTrx: Client disconnected: 192.168.1.251:55278

**Comment by pe1chl** [comment]
See issue #372.
Unfortunately it is not possible to connect between these different versions, you need to update both sides at the same time.

**Comment by sm0svx** [comment]
Yes it's not possible to connect between different versions but it should not crash of course so I'm marking this as a bug for now.

**Comment by sm3sgp** [comment]
@giulio1945 This is a very old issue and reported on an old version of svxlink. Please open a new issue if you see a crash on a later release.
